clarity/tests/arkadiko-stacker-v1-1_test.ts

```diff
@@ -325,11 +325,17 @@ Clarinet.test({
     let [stxTransferEvent1, stxTransferEvent2] = block.receipts[1].events;
     stxTransferEvent1.stx_transfer_event.sender.expectPrincipal("STSTW15D618BSZQB85R058DS46THH86YQQY6XCB7.arkadiko-stacker-v1-1");
     stxTransferEvent1.stx_transfer_event.recipient.expectPrincipal(deployer.address);
-    stxTransferEvent1.stx_transfer_event.amount.expectInt(312498000);
+
+    // TODO - Why did this change? Would have expected it to remain the same
```

Comment:
@nieldeckx you change the STX from 1000 to 1500 collateral..

since we sold off less collateral, we have more leftover (~543, see line 306). As a result, the liquidators bought less in auctions, and thus are eligible for less STX from PoX pro-rata as well.

You raised the collateral with 1/3rd (1000 -> 1500STX). The dump in price stayed the same ($2 -> $1.5)... so we sell of 1/3rd less collateral as well, and stackers get 1/3rd less collateral (312498000-(312498000/3) = 208332000)

Reply:
Alright thx for the explanation.